lib/Configuration.php

```diff
@@ -12,6 +12,9 @@ class Configuration
     /** @var bool */
     private $abortOnError = true;
 
+    /** @var bool */
+    private $throwExceptionOnInvalidFileInclude = false;
```

Comment:
I will remove it. I was thinking originally that I might need it because the Doctrine docs might have bad includes sprinkled all over the place but I can just fix all of them before upgrading doctrine/rst-parser in doctrine/doctrine-website.

Reply:
Fixed. And yes, lots of pre-existing weirdness. I plan on slowly working out the weirdness.